ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/connected/relations/TileFragmentRelationsAttributer.java

```diff
@@ -33,8 +33,10 @@ private List<LdesFragment> getCreatedTiles(List<TileFragment> tileFragments) {
 	}
 
 	private void addRelationsFromRootToCreatedTiles(LdesFragment tileRootFragment, List<LdesFragment> tileFragments) {
-		tileFragments.forEach(
-				ldesFragment -> relationsAttributer.addRelationToParentFragment(tileRootFragment, ldesFragment));
+		tileFragments.stream()
```

Comment:
have you benchmarked this ? parallel streams are not necessarily faster.

Reply:
It's a bit risky, but I think in this case it's okay to do so because the saving of the `tileRootFragment` is done after the parallel stream.